### DISCUSSION TITLE
可以做到「一组特定网域走A出口，另一组特定网域走B出口，其他直连」的效果吗

### DISCUSSION THREAD

**Discussion by abceleung** [2dust/v2rayNG#3499] [Q&A]
如题
我想达成以下效果：
1. 如浏览openai.com/chatgpt.com，则走Wireguard出口A
2. 如浏览192.168.0.0/16，则走Wireguard出口B
3. 其他全部直连

我现在的做法是加一个Wireguard出口(上方的出口A),然后在设定:
1. 域名策略：AsIs
2. 预定义规则：全局直连
3. 自定义规则（代理的网址或IP）：`domain:openai.com,domain:chatgpt.com,domain:claude.ai,domain:anthropic.com`

这样去openai.com就会用到Wireguard,其他全部直连.
我想同时做到浏览192.168.0.0/16时走另一个Wireguard出口, 请问用v2ray有方法做到吗?
谢谢

**Comment by 2dust** [comment]
用自定义配置可以实现，但是会比较难写
建议用clash

**Reply by abceleung** [reply]
> 用自定义配置可以实现，但是会比较难写 建议用clash

谢回答，不过我在网上找到很多版本的Clash，不敢乱下载，哈哈
请问有你所说的自定义配置之格式的文档吗？谢谢。

另外想问，不知道sing-box能不能做到我想要的效果？如果可以，我去看看他们的文档
